### PR TITLE
chore - add migration files for ci/cd

### PR DESCRIPTION
**Pull request by marius-hernandez**
Add liquibase for migration and refactor backend code to include core folder

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -37,15 +37,16 @@ For prod: `API_URL: https://frames-nest.onrender.com/api/`
 ```
 ### Checklist:
 - API key on header
-- Request body (case sensitive) - firstName, middleName, lastName, srCode(v1) schoolId(v2), department, program, encoding
+- Request body (case sensitive) - first_name, middle_name, last_name, srCode(v1) school_id(v2), department, program, encoding
```
Comment (AJSahagun) [inline backend/README.md:40]: Question: What is the reason for the variable name change?

**Review by AJSahagun — DISMISSED** [review]
Nice nice, thanks for the codes.

**Review by AJSahagun — APPROVED** [review]
lgtm

**Review by AJSahagun — APPROVED** [review]
lgtm 2

**Review by jkgnc — APPROVED** [review]

**Pull request merged**